CHANGELOG_API.md

```diff
@@ -0,0 +1,54 @@
+## API v0 alpha
+
+On 2023-03-02 endpoints were:
+
+- "All products" (`/api/all.json`) : Return a list of all products.
+- "Product" (`/api/{product}.json`) : Get EoL dates of all cycles of a given product.
+- "Cycle" (`/api/{product}/{cycle}.json`) : Details of a single release cycle of a given product.
+
+## API v1.0.0-b1
+
+### Summary
+
+API v1 is a major rework of the API v0 with a lot of breaking changes. Compared the API v0, API v1:
+
+- is generated using a Jekyll Generator (see https://jekyllrb.com/docs/plugins/generators/),
+- is versioned using the `api/v1` prefix, making it easier to implement non-backward-compatible
+  changes,
+- feels more "Restful".
```

Comment:
Reorder. The Jekyll implementation details are less interesting to end users than the versioning:

```suggestion
- is versioned using the `api/v1` prefix, making it easier to implement non-backward-compatible
  changes,
- feels more "RESTful".
- is generated using a Jekyll Generator (see https://jekyllrb.com/docs/plugins/generators/),
```

Reply:
Done.